### PR TITLE
Add CMS version question to GitHub issue template.

### PR DESCRIPTION
**Pull request by tech4him1**
**- Summary**

It is often helpful to know the CMS version when a user is reporting an issue, especially not that we are releasing Beta versions.

**- Test plan**
N/A

**- Description for the changelog**

Add CMS version question to GitHub issue template.

**- A picture of a cute animal (not mandatory but encouraged)**
[teaksu-kim-226783](https://unsplash.com/photos/mTkKYzlAAMc)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -27,4 +27,6 @@ You do NOT have to include this information if this is a FEATURE REQUEST
 
 **- What is the expected behavior?**
 
+**- What version of the CMS are you using?**
```
Comment (tech4him1) [inline .github/ISSUE_TEMPLATE.md]: Does anyone care if this is a separate question or combined with the one below?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -27,4 +27,4 @@ You do NOT have to include this information if this is a FEATURE REQUEST
 
 **- What is the expected behavior?**
 
-**- Please mention your node.js, and operating system version.**
+**- Please mention your CMS, node.js, and operating system version.**
```
Comment (tech4him1) [inline .github/ISSUE_TEMPLATE.md]: Does anyone care if this is a separate question or combined?

Comment (Benaiah) [reply .github/ISSUE_TEMPLATE.md]: Combined seems fine to me.

**Comment by tech4him1** [comment]
@Benaiah Done.

**Comment by Benaiah** [comment]
@tech4him1 great, thanks!

**Comment by tech4him1** [comment]
OK, squash done, should be ready to merge.

**Pull request merged**